### PR TITLE
1.0.0 beta channel

### PR DESCRIPTION
**Pull request by siddharthkp**
Green field breaking changes go here. 

~~Current version: `1.0.0-beta.2`~~ Moved to https://github.com/siddharthkp/bundlesize2#migration-from-bundlesize0180

#### This channel is not feature complete yet.

### CODE REVIEW

File: src/reporters/cli.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      printRow(file, row, maxFileLength)
+    })
+  })
+
+  printSummary(results.counter)
+}
+
+module.exports = { report }
+
+function getMaxFileLenght(files) {
```
Comment (smaclell) [inline src/reporters/cli.js:26]: Looks like a minor typo here:

```suggestion
function getMaxFileLength(files) {
```

Comment (siddharthkp) [reply src/reporters/cli.js:26]: You also gotta fix it at the site of usage 😄

File: src/reporters/cli.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const figures = require('figures')
+const bytes = require('bytes')
+const plur = require('plur')
+
+const colors = require('../utils/colors')
+
+function report(results) {
+  const files = results.files
+
+  const maxFileLength = getMaxFileLenght(files)
```
Comment (smaclell) [inline src/reporters/cli.js:11]:
```suggestion
  const maxFileLength = getMaxFileLength(files)
```

**Comment by siddharthkp** [comment]
Moved to https://github.com/siddharthkp/bundlesize2

**Pull request closed without merging**